src/os/ui/text/tuieditorlang.js

```diff
@@ -8,24 +8,22 @@ goog.provide('os.ui.text.TuiEditorLang');
  */
 os.ui.text.TuiEditorLang.setup = function(opt_supportHotkeys) {
   // This is the language block for tui editor so we can modify words in their UI
-  var options = tui.Editor.i18n._langs.get('en_US');
+  const languageOverrides = {
+    'Markdown': 'Text',
+    'WYSIWYG': 'Visual'
+  };
 
-  // Add hotkeys to tooltips
   if (opt_supportHotkeys) {
-    options['Bold'] = options['Bold'] + ' (ctrl + b)';
-    options['Blockquote'] = options['Blockquote'] + ' (alt + q)';
-    options['Code'] = options['Code'] + ' (shift + ctrl + c)';
-    // Opened issue #581
-    // options['Insert CodeBlock'] = options['Insert CodeBlock'] + ' (shift + ctrl + p)';
-    options['Italic'] = options['Italic'] + ' (ctrl + i)';
-    options['Line'] = options['Line'] + ' (ctrl + l)';
-    options['Ordered list'] = options['Ordered list'] + ' (ctrl + o)';
-    options['Strike'] = options['Strike'] + ' (ctrl + s)';
-    options['Task'] = options['Task'] + ' (alt + t)';
-    options['Unordered list'] = options['Unordered list'] + ' (ctrl + u)';
+    languageOverrides['Bold'] = 'Bold (ctrl + b)';
+    languageOverrides['Blockquote'] = 'Blockquote (alt + q)';
+    languageOverrides['Code'] = 'Inline code (shift + ctrl + c)';
```

Comment:
Do we need an override for the code block too?
I know it was commented out in the old version but may be fixed in the new version? idk

Reply:
It hasn't changed for the new version, so I would leave the language override out. While the shortcut still works in Chrome it still does _not_ work in Firefox.